docs/quick-start-guide.md

```diff
@@ -95,7 +95,7 @@ STORED AS carbondata;
 ###### Loading Data to a Table
 
 ```
-LOAD DATA INPATH '/local-path/sample.csv' INTO TABLE test_table;
+LOAD DATA INPATH 'file:///local-path/sample.csv' INTO TABLE test_table;
```

Comment:
Here ,only give the example, need to give the actual address when user using it.